packages/graphql/src/schema.ts

```diff
@@ -8,8 +8,11 @@ import { mutationErrorPlugin, nexusDebugLogPlugin, nexusSlowGuardPlugin, nexusDe
 
 const isCodegen = Boolean(process.env.CYPRESS_INTERNAL_NEXUS_CODEGEN)
 
+// TODO: fix this with an update to esbuild
```

Comment:
Is this esbuild fix coming before this feature branch merges? I little more context might be helpful here if not.

Reply:
Added context: [cypress-io/cypress@`ec34a53` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/ec34a537a4879edcfbc0409b0f81f16a94e97ee9)